cluster/ci/config/prow/prow_images.yaml

```diff
@@ -0,0 +1,213 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: prow-images
+objects:
+- kind: ImageStream
+  apiVersion: v1
+  metadata:
+    name: prow
+- kind: BuildConfig
+  apiVersion: v1
+  metadata:
+    name: prow
+  spec:
+    output:
+      to:
+        kind: ImageStreamTag
+        name: prow:binaries
+    source:
+      git:
+        uri: https://github.com/openshift/test-infra.git
+      type: Git
+      dockerfile: |
+        FROM openshift/origin-release:golang-1.8
+
+        COPY . /go/src/k8s.io/test-infra
+        RUN cd /go/src/k8s.io/test-infra && \
+            godep restore && \
+            cd prow && \
+            mkdir -p /static && \
+            cp -r cmd/deck/static/* /static && \
+            CGO_ENABLED=0 go install -a -installsuffix cgo -ldflags '-w' ./cmd/... && \
```

Comment:
Eventually, each component should have its own build chain. I am thinking of fixing it in this PR..